### DISCUSSION TITLE
How to not logging for certain endpoints?

### DISCUSSION THREAD

**Discussion by ELymar** [ThreeMammals/Ocelot#1891] [Q&A]
## Expected Behavior / New Feature
Could it be possible to give more control over what gets logged? We have API keys that get passed in as query parameters, and the logger ends up storing them in plaintext in our logs. Could we disable logging query parameters, or perhaps just not logging for certain endpoints? If there is a workaround (besides passing keys in the header) I'd also appreciate suggestions.

## Actual Behavior / Motivation for New Feature
Currently by adding a logger via DI, all requests that go through the system are logged along with their query parameters. 

## Steps to Reproduce the Problem
N/A
  

## Specifications

  - Version: latest
  - Platform: Azure Service Fabric
  - Subsystem:

**Comment by raman-m** [comment]
Hello @ELymar !
I don't understand your user scenarios...
Why don't you want to log some routes URLs but the rest of them should be?
Just increase log level to `Error` and your problem is over. 😉 

Regarding design.
I don't see much sense to develop custom logger in Ocelot to filter query strings. Better to increase log level or/and remove sensitive information from URLs at all, by switching to auth tokens, for example.
@ggnaegi  What do you think?
Your recommendation?
